provider/README.md

```diff
@@ -199,14 +201,17 @@ Schema: [`trips` schema][trips-schema]
 
 ### Trips Query Parameters
 
-The trips API should allow querying trips with a combination of query parameters.
+The trips API should allow querying trips with the following query parameters. 
 
-* `device_id`
-* `vehicle_id`
-* `min_end_time`: filters for trips where `end_time` occurs at or after the given time
-* `max_end_time`: filters for trips where `end_time` occurs before the given time
+Without an `end_time` query parameter, `/trips` shall return an error. 
 
-When multiple query parameters are specified, they should all apply to the returned trips. For example, a request with `?min_end_time=1549800000000&max_end_time=1549886400000` should only return trips whose end time falls in the range `[1549800000000, 1549886400000)`.
+* `end_time`: An [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date for a static bucket of files representing an Hour. The response will have all trips whose end time occurred in the hour. For example, if you requested "2019-10-01:07", you should get back all trips that ended between `7:00 <= and < 8:00`. Timezone UTC. 
+
+| Request range | format | expected output | 
+| ------------- | ------ | --------------- | 
+| Hour          | YYYY-MM-DD:HH | That hour worth of trips, or if the hour is not complete or the file does not exist, an error. The minimum value of the hour should be 00 and the max 23, as the midnight hour should be denoted as the next day's zero hour. | 
```

Comment:
That's a `T`, not a `:` in ISO 8601 definition. https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations

Reply:
@hyperknot- Thanks for catching that! fixed in another commit.